messenger/messenger.go

```diff
@@ -77,13 +94,15 @@ func (a *App) makeExecutionServer() *ExecutionServiceServerV1Alpha2 {
 func (a *App) setupRestRoutes() {
 	a.restRouter.HandleFunc("/block/{height}", a.getBlock).Methods("GET")
 	a.restRouter.HandleFunc("/message", a.postMessage).Methods("POST")
+	a.restRouter.HandleFunc("/recent", a.getRecentMessages).Methods("GET")
+	a.restRouter.HandleFunc("/ws", a.serveWS)
```

Comment:
from a practical perspective (to get 8bit.chat running) i see why adding websockets was helpful here. but from the perspective of making a simple, digestible demo i think that sticking with post and get endpoints would be clearer. not that i think this specifically needs to change, but something to keep in mind when we make this into a tutorial, maybe?

Reply:
yeah that is fair, this was a ship it in half a day approach. It is still using the `/message` endpoint to post new messages. But is using the websocket to stream new messages as they come in. For a tutorial it should either be using all rest or all websockets and not mixing them. for a chat app, I think websockets makes more sense, but from a digestible tutorial point of view, rest is simpler since you don't have to deal with any of the client connection management